docs/garmin-xhd-output.md

```
@@ -0,0 +1,136 @@
+- Mayara built with the feature (it is not in the standard binaries):
+
+  ```sh
+  cargo build --release --features garmin-xhd-output
+  ```
+
+- An address on the Garmin Marine Network, `172.16.x.x` with netmask
+  `255.240.0.0`, on the interface the plotter is reachable through. If the
+  radar is on a different subnet — a Furuno on `172.31.x.x`, say — give the
+  same interface both addresses:
+
+  ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)